libnavigation-core/src/test/java/com/mapbox/navigation/core/internal/trip/session/MapboxTripSessionTest.kt

```diff
@@ -100,7 +100,8 @@ class MapboxTripSessionTest {
             locationEngine,
             navigatorPredictionMillis,
             navigator,
-            logger = logger
+            logger = logger,
+            accessToken = "pk.1234"
```

Comment:
same question as above in the `MapboxTripSession`.
If the `accessToken` is possible to be **NULL**, can we add a test case to cover the expected behavior here?

Reply:
We cannot add a unit test case in case the token is NULL. This would be better covered with instrumentation test, as if the token is null, you would see no guidance view being drawn and we could have verify that. But, in this case there is nothing to verify from unit test point of view.

Reply:
We could verify the URL structure without a token.

Reply:
Also would be great to add a test that verifies the behavior if there is a token.